### ISSUE TITLE
Unsatisfiable dependencies for conda install on Windows

### ISSUE THREAD

**Issue by dhslichter** [m-labs/artiq#255]
Installing conda packages on Windows -- some clauses restricting dependencies appear to be unsatisfiable.  Output is below:
```
C:\Users\rabi>conda create -n artiq-2016-02-03 artiq=0.1=py_798+gitb7de92e --alt
-hint
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ........
Solving package specifications: ................................
Error: Unsatisfiable package specifications.
Generating minimal hint:
[      COMPLETE      ]|##################################################| 100%

The following set of clauses is unsatisfiable:

not qt-4.8.7-vc14_4
not qt-4.8.7-vc10_4
not qt-4.8.7-vc9_5
artiq-0.1-py_798+gitb7de92e
not qt-4.8.7-vc10_5
artiq-0.1-py_798+gitb7de92e => quamash-0.5.1-py35_4 or quamash-0.5.1-py_5
not qt-4.8.7-vc9_4
quamash-0.5.1-py_5 => pyqt-4.11.4-py35_2 or pyqt-4.11.4-py35_3 or pyqt-4.11.4-py
35_4
quamash-0.5.1-py35_4 => pyqt-4.11.4-py35_2 or pyqt-4.11.4-py35_3 or pyqt-4.11.4-
py35_4
not qt-4.8.7-vc14_5
pyqt-4.11.4-py35_2 => qt-4.8.7-vc10_4 or qt-4.8.7-vc9_4 or qt-4.8.7-vc9_5 or qt-
4.8.7-vc14_5 or qt-4.8.7-vc10_5 or qt-4.8.7-vc14_4
pyqt-4.11.4-py35_3 => qt-4.8.7-vc10_4 or qt-4.8.7-vc9_4 or qt-4.8.7-vc9_5 or qt-
4.8.7-vc14_5 or qt-4.8.7-vc10_5 or qt-4.8.7-vc14_4
pyqt-4.11.4-py35_4 => qt-4.8.7-vc10_4 or qt-4.8.7-vc9_4 or qt-4.8.7-vc9_5 or qt-
4.8.7-vc14_5 or qt-4.8.7-vc10_5 or qt-4.8.7-vc14_4

```
Here is the info for the current conda install:
```
C:\Users\rabi>conda info
Using Anaconda Cloud api site https://api.anaconda.org
Current conda install:

             platform : win-32
        conda version : 3.19.1
  conda-build version : 1.19.0
       python version : 3.5.1.final.0
     requests version : 2.9.1
     root environment : C:\Anaconda3  (writable)
  default environment : C:\Anaconda3
     envs directories : C:\Anaconda3\envs
        package cache : C:\Anaconda3\pkgs
         channel URLs : http://conda.anaconda.org/m-labs/channel/main/win-32/
                        http://conda.anaconda.org/m-labs/channel/main/noarch/
                        http://conda.anaconda.org/m-labs/channel/dev/win-32/
                        http://conda.anaconda.org/m-labs/channel/dev/noarch/
                        https://repo.continuum.io/pkgs/free/win-32/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-32/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : C:\Users\rabi\.condarc
    is foreign system : False
```

Here is a list of the installed packages:

```

C:\Users\rabi>conda list
# packages in environment at C:\Anaconda3:
#
_license                  1.1                      py35_1
alabaster                 0.7.7                    py35_0
anaconda                  2.4.1               np110py35_0
anaconda-client           1.2.2                    py35_0
argcomplete               1.0.0                    py35_1
astropy                   1.1.1               np110py35_0
babel                     2.2.0                    py35_0
beautifulsoup4            4.4.1                    py35_0
bitarray                  0.8.1                    py35_1
blaze                     0.9.0                     <pip>
blaze-core                0.9.0                    py35_0
bokeh                     0.11.0                   py35_1
boto                      2.39.0                   py35_0
bottleneck                1.0.0               np110py35_0
bzip2                     1.0.6                    vc14_2  [vc14]
cffi                      1.2.1                    py35_0
clyent                    1.2.0                    py35_0
colorama                  0.3.6                    py35_0
comtypes                  1.1.2                    py35_0
conda                     3.19.1                   py35_0
conda-build               1.19.0                   py35_0
conda-env                 2.4.5                    py35_0
configobj                 5.0.6                    py35_0
console_shortcut          0.1.1                    py35_1
cryptography              1.0.2                    py35_0
curl                      7.45.0                   vc14_1  [vc14]
cycler                    0.9.0                    py35_0
cython                    0.23.4                   py35_0
cytoolz                   0.7.5                    py35_0
datashape                 0.5.0                    py35_0
decorator                 4.0.6                    py35_0
docutils                  0.12                     py35_1
dynd                      f641248                   <pip>
dynd-python               0.7.1                    py35_0
et-xmlfile                1.0.1                     <pip>
et_xmlfile                1.0.1                    py35_0
fastcache                 1.0.2                    py35_0
flask                     0.10.1                   py35_1
freetype                  2.5.5                    vc14_0  [vc14]
greenlet                  0.4.9                    py35_0
h5py                      2.5.0               np110py35_4
hdf5                      1.8.15.1                 vc14_4  [vc14]
idna                      2.0                      py35_0
ipykernel                 4.2.2                    py35_0
ipython                   4.0.3                    py35_0
ipython-genutils          0.1.0                     <pip>
ipython-notebook          4.0.4                    py35_3
ipython-qtconsole         4.0.1                    py35_4
ipython_genutils          0.1.0                    py35_0
ipywidgets                4.1.1                    py35_0
itsdangerous              0.24                     py35_0
jdcal                     1.2                      py35_0
jedi                      0.9.0                    py35_0
jinja2                    2.8                      py35_0
jpeg                      8d                       vc14_0  [vc14]
jsonschema                2.4.0                    py35_0
jupyter                   1.0.0                    py35_1
jupyter-client            4.1.1                     <pip>
jupyter-console           4.1.0                     <pip>
jupyter-core              4.0.6                     <pip>
jupyter_client            4.1.1                    py35_0
jupyter_console           4.1.0                    py35_0
jupyter_core              4.0.6                    py35_0
launcher                  1.0.0                         4
libdynd                   0.7.1                         0
libpng                    1.6.17                   vc14_1  [vc14]
libsodium                 1.0.3                         0
libtiff                   4.0.6                    vc14_1  [vc14]
llvmlite                  0.8.0                    py35_0
lxml                      3.5.0                    py35_0
markupsafe                0.23                     py35_0
matplotlib                1.5.1               np110py35_0
menuinst                  1.3.2                    py35_0
mistune                   0.7.1                    py35_0
msvc_runtime              1.0.1                    vc14_0  [vc14]
multipledispatch          0.4.8                    py35_0
nbconvert                 4.1.0                    py35_0
nbformat                  4.0.1                    py35_0
networkx                  1.11                     py35_0
nltk                      3.1                      py35_0
node-webkit               0.10.1                        0
nose                      1.3.7                    py35_0
notebook                  4.1.0                    py35_0
numba                     0.23.1              np110py35_0
numexpr                   2.4.6               np110py35_0
numpy                     1.10.1                   py35_0
odo                       0.4.0                    py35_0
openpyxl                  2.3.2                    py35_0
openssl                   1.0.2f                   vc14_0  [vc14]
pandas                    0.17.1              np110py35_0
patch                     2.5.9                         1
path.py                   8.1.2                    py35_1
patsy                     0.4.0               np110py35_0
pep8                      1.7.0                    py35_0
pickleshare               0.5                      py35_0
pillow                    3.1.0                    py35_0
pip                       8.0.2                    py35_0
ply                       3.8                      py35_0
psutil                    3.4.2                    py35_0
py                        1.4.31                   py35_0
pyasn1                    0.1.9                    py35_0
pycosat                   0.6.1                    py35_0
pycparser                 2.14                     py35_0
pycrypto                  2.6.1                    py35_3
pycurl                    7.19.5.3                 py35_1
pyflakes                  1.0.0                    py35_0
pygments                  2.1                      py35_0
pyopenssl                 0.15.1                   py35_1
pyparsing                 2.0.3                    py35_0
pyqt                      4.11.4                   py35_4
pyreadline                2.1                      py35_0
pytables                  3.2.2               np110py35_1
pytest                    2.8.5                    py35_0
python                    3.5.1                         2
python-dateutil           2.4.2                    py35_0
pytz                      2015.7                   py35_0
pywin32                   219                      py35_2
pyyaml                    3.11                     py35_2
pyzmq                     15.2.0                   py35_0
qt                        4.8.7                    vc14_5  [vc14]
qtconsole                 4.1.1                    py35_0
requests                  2.9.1                    py35_0
rope                      0.9.4                    py35_1
rope-py3k                 0.9.4.post1               <pip>
scikit-image              0.11.3              np110py35_0
scikit-learn              0.17                np110py35_1
scipy                     0.16.0              np110py35_0
setuptools                19.6.2                   py35_0
simplegeneric             0.8.1                    py35_0
sip                       4.16.9                   py35_2
six                       1.10.0                   py35_0
snowballstemmer           1.2.1                    py35_0
sockjs-tornado            1.0.1                    py35_0
sphinx                    1.3.5                    py35_0
sphinx-rtd-theme          0.1.9                     <pip>
sphinx_rtd_theme          0.1.9                    py35_0
spyder                    2.3.8                    py35_0
spyder-app                2.3.8                    py35_0
sqlalchemy                1.0.11                   py35_0
statsmodels               0.6.1               np110py35_0
sympy                     0.7.6.1                  py35_0
tables                    3.2.2                     <pip>
tk                        8.5.18                   vc14_0  [vc14]
toolz                     0.7.4                    py35_0
tornado                   4.3                      py35_0
traitlets                 4.1.0                    py35_0
ujson                     1.35                     py35_0
unicodecsv                0.14.1                   py35_0
werkzeug                  0.11.3                   py35_0
wheel                     0.26.0                   py35_1
xlrd                      0.9.4                    py35_0
xlsxwriter                0.8.4                    py35_0
xlwings                   0.6.4                    py35_1
xlwt                      1.0.0                    py35_0
zeromq                    4.1.3                    vc14_1  [vc14]
zlib                      1.2.8                    vc14_2  [vc14]
```

**Comment by dhslichter** [comment]
This issue is fixed if one first creates an environment for python 3.5, and then installs artiq into that environment:

```
C:\users\rabi> conda create --name artiq-2016-02-03 python=3.5
C:\users\rabi> activate artiq-2016-02-03
[artiq-2016-02-03] C:\Users\rabi>conda install artiq=0.1=py_798+gitb7de92e
```

This was the fix before for issue #133 , but I am surprised because this is a new Anaconda install with Python 3.5 only.  This issue does not appear when doing a conda install on Linux.  Perhaps something with dependencies in the win32 version of the quamash package?

**Comment by sbourdeauducq** [comment]
We are going to switch to Qt5, so whatever the pyqt/quamash packages are doing wrong should become irrelevant.
